jjb/anax/Jenkinsfile

```diff
@@ -0,0 +1,35 @@
+pipeline {
```

Comment:
The pipeline should be stored in the project repo, not in the ci-management repo.